lib/Utils/NetworkHelper.pm

```diff
@@ -0,0 +1,29 @@
+package Utils::NetworkHelper;
+
+use mm_network qw(configure_default_gateway configure_static_ip configure_static_dns get_host_resolv_conf);
+use Exporter 'import';
+use strict;
+use warnings;
+
+our @EXPORT            = qw(setup_static_mm_network %setup_nis_nfs_x11);
+our %setup_nis_nfs_x11 = (
```

Comment:
This hash map is input data for nis tests, I don't think it belongs to the lib file. I don't have good proposal where we should store data for the tests right now, so maybe we can leave this for later and keep it in `lib/y2_guitest.pm`? WDYT?